chapters/introduction.tex

```diff
@@ -99,14 +98,62 @@ \section{Some Definitions}\label{some-definitions}
 % - quality of implementation
 % - simulator
 
-\section{Notation and Grammar}\label{notation-and-grammar}
-
-The meta symbols (of the extended BNF-grammar) are defined in~\cref{lexical-conventions}.
-
-Boldface denotes keywords of the Modelica language. Keywords are
-reserved words and shall not be used as identifiers, with the exception of
-\lstinline!initial! which is a keyword in section headings, and \lstinline!der! which is a
-keyword for declaration functions, but it is also possible to call the
-functions \lstinline!initial! and \lstinline!der!.
+\section{Notation}\label{notation}
+
+The remainder of this section shows examples of the presentation used in this document.
+
+Syntax highlighting of Modelica code is illustrated by the code listing below.
+Things to note include keywords that define code structure such as \lstinline!equation!, keywords that do not define code structure such as \lstinline!connect!, and recognized identifiers with meaning defined by the specification such as \lstinline!semiLinear!:
+\begin{lstlisting}[language=modelica]
+model Example "Example used to illustrate syntax highlighting"
+  /* The string above is a class description string, this is a comment. */
+//String s = 1.0; /* Commenting out invalid code like this is rare. */
+  /* Instead of commenting-out invalid code, it is typically done this way: */
+  String s = 1.0; // Error: No conversion form Real to String.
```

Comment:
I find these comments about invalid code unclear, and I don't think it is needed. The same for "Instead of ..." on the next line.

Just define that we normally (or even always?) handle invalid code, by having it and describing the issue.
```suggestion
  /* Invalid code is typically done this way: */
  String s = 1.0; // Error: No conversion form Real to String.
```